src/firefly/js/util/VOAnalyzer.js

```diff
@@ -416,8 +416,8 @@ class TableRecognizer {
             let lonCol;
             let latCol;
             if (useReg) {
-                const reLon= new RegExp(`^[A-z]?[-_]?(${lon})[1-9]*$`);
-                const reLat= new RegExp(`^[A-z]?[-_]?(${lat})[1-9]*$`);
+                const reLon= new RegExp(`^[A-z]*?[-_]?(${lon})[1-9]*$`);
+                const reLat= new RegExp(`^[A-z]*?[-_]?(${lat})[1-9]*$`);
```

Comment:
You probably mean to replace `?`(0 or 1) with `*`(0 or more).  Not both.

Reply:
ok, your right. thanks